### PR TITLE
Include datapack.unity3d asset bundle

### PR DESCRIPTION
**Pull request by victoraldecoa**
Add missing datapack.unity3d asset bundle as a compressed game file

### CODE REVIEW

**Comment by ds5678** [comment]
How can I produce a game using that bundle name?

**Comment by victoraldecoa** [comment]
According to the Unity source code, this bundle is present in Universal Android Apks.

ApkFile.cpp:268
```cpp
// This searches for the data archive in the provided path and mounts it if it exists
// If the provided path is to the "Unity data asset pack", then there should always be only datapack.unity3d archive.
// If the provided path is to the base apk, then there should alwyas be data.unity3d archive, but in case we are running
// an universal APK, then datapack.unity3d could also be present in the base apk.
bool MountDataArchive(const core::string& path)
{
    if (!FileSystemIsMounted())
        return false;

    const core::string dataSubPath("assets/bin/Data");
    const core::string alternativeDataBundleFilename("datapack.unity3d");
```

**Comment by victoraldecoa** [comment]
**tl;dr** Build AAB with Gradle with Unity data asset pack inside and then convert the AAB into universal APK using bundletool.

**Comment by victoraldecoa** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by victoraldecoa** [comment]
recheck

**Comment by ds5678** [comment]
Can you add a link to the source code in your changes?

**Comment by victoraldecoa** [comment]
I not sure if I understood your question, but Unity's source code is closed for enterprise use only. I could add a comment explaining how to produce a build with that asset as I did above, wdyt?

**Comment by victoraldecoa** [comment]
Ptal: 102281cca
